.github/workflows/package.yml

```diff
@@ -0,0 +1,114 @@
+name: Package
+
+on:
+  push:
+    tags:
+      - v*.*.*
+
+jobs:
+
+  wheel-linux:
+    runs-on: ubuntu-latest
+    name: Build Linux wheels
+    strategy:
+      matrix:
+        include:
+        - python-version: 3.6
+          pyo3-python: /opt/python/cp36-cp36m/bin/python
+        - python-version: 3.7
+          pyo3-python: /opt/python/cp37-cp37m/bin/python
+        - python-version: 3.8
+          pyo3-python: /opt/python/cp38-cp38/bin/python
+        - python-version: 3.9
+          pyo3-python: /opt/python/cp39-cp39/bin/python
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+      with:
+        submodules: true
+    - name: Build manylinux wheels
+      uses: addnab/docker-run-action@v2
+      with:
+        image: pypywheels/manylinux2010-pypy_x86_64
+        options: -v ${{ github.workspace }}:/io -e TERM=$TERM
+        shell: bash
+        run: |
+          ${{ matrix.pyo3-python }} -m pip install -U wheel auditwheel
+          ${{ matrix.pyo3-python }} -m pip wheel --no-deps -vv /io -w /tmp
+          for whl in /tmp/*.whl; do ${{ matrix.pyo3-python }} -m auditwheel repair $whl -w /io/dist; done
+    - name: Store built wheels
+      uses: actions/upload-artifact@v2
+      with:
+        name: dist
+        path: dist/*
+
+  wheel-osx:
+    runs-on: macos-latest
+    name: Build OSX wheels
+    strategy:
+      matrix:
+        python-version:
+        - 3.6
+        - 3.7
+        - 3.8
+        - 3.9
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+      with:
+        submodules: true
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install Python requirements
+      run: python -m pip install -r ci/requirements.txt
+    - name: Build wheel distribution
+      run: python setup.py bdist_wheel
+    - name: Store built wheels
+      uses: actions/upload-artifact@v2
+      with:
+        name: dist
+        path: dist/*
+
+  sdist:
+    runs-on: ubuntu-latest
+    name: Build source distribution
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+      with:
+        submodules: true
+    - name: Set up Python 3.9
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
+    - name: Build source distribution
+      run: python setup.py sdist
+    - name: Store built wheels
+      uses: actions/upload-artifact@v2
+      with:
+        name: dist
+        path: dist/*
+
+  upload:
+    environment: PyPI
+    runs-on: ubuntu-latest
+    name: Upload
+    needs:
+    - sdist
+    - wheel-linux
+    - wheel-osx
+    steps:
+    - name: Download built distributions
+      uses: actions/download-artifact@v2
+      with:
+          name: dist
+          path: dist
+    - name: Publish distributions to PyPI
+      if: startsWith(github.ref, 'refs/tags/v')
```

Comment:
I updated the condition for the entire `package.yml` workflow to be built, basically now it will only run for tags matching the release format you specified above. 

Unfortunately, Actions handle tags differently from Travis-CI, so a workflow can be triggered by a branch or a tag, but not both. This means you cannot know if a tag was on the `master` branch or somewhere else.